kube/kd/deployment.yaml

```diff
@@ -0,0 +1,222 @@
+apiVersion: apps/v1
```

Comment:
You've changed enough of this file that Git doesn't automatically recognise it as a rename: use `git mv` instead of `mv` in future to explicitly tell it what's going on -- otherwise you'll lose the history.

Reply:
I did a `git mv [old] [new]`, not sure why it's not showing as a move.